db/schema.rb

```diff
@@ -133,6 +133,8 @@
     t.string "description"
     t.string "gender"
     t.string "training_style"
+    t.float "latitude"
```

Comment:
are we meant to have these two lines this time? last time, we had to remove these two lines manually.

Reply:
Yes! This time it's normal to have them since there's a migration creating them 👍